<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="panal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>501</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="memoryTab">
     <attribute name="title">
      <string>memory</string>
     </attribute>
     <widget class="QGroupBox" name="CPU">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>CPU</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>160</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="CPUUsageLabel">
          <property name="text">
           <string>CPU Usage: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="CPUUsageValue">
          <property name="text">
           <string>value</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGraphicsView" name="cpuChartView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>461</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="memAndSwp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>481</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>memory and swap</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>431</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>memory</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QProgressBar" name="memBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QProgressBar" name="swpBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>swap</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="memDetailValue">
          <property name="text">
           <string>value</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="swpDetailValue">
          <property name="text">
           <string>value</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="processTab">
     <attribute name="title">
      <string>process</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>311</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>479</width>
         <height>309</height>
        </rect>
       </property>
       <widget class="QTableView" name="taskTable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>461</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>terminate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>330</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>fresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="taskDetailValue">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
     <widget class="QLabel" name="searchModeLabel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>330</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Search Mode</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="systemTab">
     <attribute name="title">
      <string>system</string>
     </attribute>
     <widget class="QGroupBox" name="CpuInfoGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>CPU</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>461</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>CPU: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="cpuNameValue">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Cache: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="cacheValue">
          <property name="text">
           <string>value</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="OsInfoGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>481</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Operation System</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>461</width>
         <height>44</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="osTypeValue">
            <property name="text">
             <string>linux</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="osReleaseValue">
            <property name="text">
             <string>xxx</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="hostNameGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>481</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Hostname</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>461</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="hostnameLabel">
          <property name="text">
           <string>Hostname: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hostnameValue">
          <property name="text">
           <string>value</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="DatetimeGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>481</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>DateTime</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>461</width>
         <height>42</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Current: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="CurrentDatatimeValue">
          <property name="text">
           <string>value</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Last login: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lastLoginDatatimeValue">
          <property name="text">
           <string>value</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="helpTab">
     <attribute name="title">
      <string>help</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>reboot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>120</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>shutdown</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>170</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Hello!</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>201</width>
         <height>42</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Welcome here!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Author: li Yifan</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>231</width>
         <height>42</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>lyifan0000@gmail.com</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
